src/path.cc

```diff
@@ -135,39 +125,42 @@ std::string PathResolve(Environment* env,
 
     // Try to match a root
     if (len == 1) {
-      if (IsPathSeparator(code)) {
+      if (code == std::filesystem::path::preferred_separator) {
         // `path` contains just a path separator
         rootEnd = 1;
         isAbsolute = true;
       }
-    } else if (IsPathSeparator(code)) {
+    } else if (code == std::filesystem::path::preferred_separator) {
       // Possible UNC root
 
       // If we started with a separator, we know we at least have an
       // absolute path of some kind (UNC or otherwise)
       isAbsolute = true;
 
-      if (IsPathSeparator(path[1])) {
+      if (path[1] == std::filesystem::path::preferred_separator) {
         // Matched double path separator at beginning
         size_t j = 2;
         size_t last = j;
         // Match 1 or more non-path separators
-        while (j < len && !IsPathSeparator(path[j])) {
+        while (j < len &&
+               path[j] != std::filesystem::path::preferred_separator) {
           j++;
         }
         if (j < len && j != last) {
           const std::string firstPart = path.substr(last, j - last);
           // Matched!
           last = j;
           // Match 1 or more path separators
-          while (j < len && IsPathSeparator(path[j])) {
+          while (j < len &&
+                 path[j] == std::filesystem::path::preferred_separator) {
             j++;
           }
           if (j < len && j != last) {
             // Matched!
             last = j;
             // Match 1 or more non-path separators
-            while (j < len && !IsPathSeparator(path[j])) {
+            while (j < len &&
+                   !path[j] == std::filesystem::path::preferred_separator) {
```

Comment:
Shouldn't this be:
```suggestion
                   path[j] != std::filesystem::path::preferred_separator) {
```
I don't do cpp, but this is inconsistent with https://github.com/nodejs/node/pull/53063/files#diff-bb0465a568712d51fdd277a816d9773401bfa62d8d57c7c124326b55f8652806R146 🙂

Reply:
Yes, I just reverted this change.